CMakeLists.txt

```diff
@@ -435,7 +435,14 @@ else ()
   endif ()
   gflags_define (PATH LIBRARY_INSTALL_DIR "Directory of installed libraries, e.g., \"lib64\"" "lib${LIB_SUFFIX}")
   gflags_property (LIBRARY_INSTALL_DIR ADVANCED TRUE)
-  set (INCLUDE_INSTALL_DIR include)
+  
+  # Cross-compiling for QNX does not install in the normal include path
+  if (QNX)
+    set (INCLUDE_INSTALL_DIR ${QNX_INCLUDE_INSTALL_DIR})
+  else()
+    set (INCLUDE_INSTALL_DIR include)
+  endif ()
+
```

Comment:
I'm curious if setting `GFLAGS_INCLUDE_DIR` to the desired output path would have been sufficient?

Reply:
Hi,
I did some experimenting and found that this line isn't required with QNX's default toolchain. This was based on an internal port, so I think it's a holdover from a specific project with weird requirements. I've reverted it with 6c059177ca1b82ba54ff5b6be74349008169d9f5.